commcare-cloud/commcare_cloud/commcare_cloud.py

```diff
@@ -46,6 +45,13 @@ def arg_branch(parser):
     ))
 
 
+def arg_stdout_callback(parser, default='default'):
+    parser.add_argument(
+        '--output', dest='stdout_callback', default=default, choices=['actionable', 'minimal'],
+        help=("The callback plugin to use for generating output."
```

Comment:
Might also be worth including "See `ansible-doc -t callback -l` and `ansible-doc -t callback [ansible|minimal]` for more details.

Reply:
898076a